ops/model.py

```diff
@@ -1068,6 +1068,26 @@ def get_plan(self) -> 'pebble.Plan':
         """Get the current effective pebble configuration."""
         return self._pebble.get_plan()
 
+    def get_services(self, *service_names: str) -> typing.List['pebble.ServiceInfo']:
+        """Get a list of service status information.
+
+        If no service names are specified, return status information for all
+        services, otherwise return information for only the given services.
+        """
+        return self._pebble.get_services(service_names)
+
+    def get_service(self, service_name: str) -> 'pebble.ServiceInfo':
+        """Get status information for a single named service.
+
+        Raises model error if service_name is not found.
+        """
+        services = self.get_services(service_name)
+        if not services:
```

Comment:
I'm guessing an array of length 0 is falsey?

Reply:
Yes, that's right. And it's idiomatic in Python to use that. See [comment in PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations): "For sequences, (strings, lists, tuples), use the fact that empty sequences are false"